docs/pages/identity-governance/entra-id.mdx

```diff
@@ -0,0 +1,359 @@
+add a `groups` claim. 
+
+![Entra ID Integration](../../img/igs/entraid/entra-sso-config.png)
+
+
+## Step 3/5. Configure OIDC IdP
+
+Under “App registrations” from Azure services menu, find and open your enterprise
+application created in step 1.
+
+Select “Manage > Certificates & secrets” and then select “Federated credentials“.
+Click `+ Add credential` button.
+
+Under “Add a credential” UI, configure credential with the following values:
+- **Federated credential scenario:** Other issuer
+
+Under “Connect your account”, configure the following values:
+- **Issuer:** https://example.com
+- **Type:** Explicit subject identifier
+- **Value:** teleport-azure
+
+Under “Credential details”, configure the following values:
+- **Name:** teleport-oidc
+- **Description:** Teleport OIDC Identity Provider
+
+![Entra ID Integration](../../img/igs/entraid/entra-setup-oidc.png)
+
+## Step 4/5. Configure API permissions
+
+Under the same App registration UI for your enterprise application, select “Manage > API permissions”.
+
+You can add a new graph permission by clicking on `+ Add a permission` button and then selecting 
+“Microsoft Graph > Application permissions”.
+
+The following permissions needs to be added to the application. 
+- `Application.ReadWrite.OwnedBy`
+- `Group.Read.All`
+- `User.Read.All`
+
+![Entra ID Integration](../../img/igs/entraid/entra-api-permission.png)
+
+
+## Step 5/5. Install the Entra ID plugin
+
+Now run the `tctl plugins install entraid` command.
+
+```code
+$ tctl plugins install entraid \
+    --name entra-id-default \
+    --auth-connector-name entra-id \
+    --default-owner=<Var name="Access List Owner"/> \
+    –-manual-setup
+```
+
+`tctl` will then prompt for Entra ID tenant ID and Application ID of the enterprise 
+application created in step 1.
+
+Once you enter these values, Entra ID plugin will be installed with the OIDC IdP based authentication method.
+
+## FAQ
+
+### What resources are imported to Teleport?
+
+Teleport imports all the users, user groups and its members from the Entra ID directory.
+There is no filter available to control custom import rules.
+
+If Teleport Identity Security integration is enabled, Teleport will import 
+applications and policies as well. 
+
+### How does it work with nested Access Lists?
+
+If an Entra ID group is assigned as a member to another group, Teleport preserves this assignment 
+as a nested Access List. 
+
+However, note that Teleport does not support recursive groups where group A is a member of group B
+and group B is also a member of group A.
+
+### What permissions does Teleport need to authenticate with the Microsoft Graph API?
+
+At minimum, Teleport needs read access to users, groups and the main enterprise application 
+for which the integration is configured.
+```code 
+- Application.ReadWrite.OwnedBy
+- Group.Read.All
+- User.Read.All
+```
+
+If you enable Identity Security integration, you will need a broader sets of permission.
```

Comment:
```suggestion
If you enable the Identity Security integration, you will need a broader set of permissions.
```

Do these permissions _replace_ the ones above, or are they needed in addition to the ones above?

Reply:
Replace it. Added comment to hint that.